### PR TITLE
Revert "fix: use short sha version"

### PR DESCRIPTION
**Pull request by pedrosousa**
Reverts cloudflare/cloudflare-docs#2805
Also removes the previous long SHA from the URL.

### CODE REVIEW

File: products/email-routing/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "dependencies": {
-    "cloudflare-docs-engine": "git+https://github.com/cloudflare/cloudflare-docs-engine.git#1a796d3e"
+    "cloudflare-docs-engine": "git+https://github.com/cloudflare/cloudflare-docs-engine.git#1a796d3eaaa018da4c6ca4106bf369a2017bc4ec"
```
Comment (pedrosousa) [inline products/email-routing/package.json]:
```suggestion
    "cloudflare-docs-engine": "git+https://github.com/cloudflare/cloudflare-docs-engine.git"
```

File: products/fundamentals/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "dependencies": {
-    "cloudflare-docs-engine": "git+https://github.com/cloudflare/cloudflare-docs-engine.git#1a796d3e"
+    "cloudflare-docs-engine": "git+https://github.com/cloudflare/cloudflare-docs-engine.git#1a796d3eaaa018da4c6ca4106bf369a2017bc4ec"
```
Comment (pedrosousa) [inline products/fundamentals/package.json]:
```suggestion
    "cloudflare-docs-engine": "git+https://github.com/cloudflare/cloudflare-docs-engine.git"
```

File: products/workers/package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "dependencies": {
-    "cloudflare-docs-engine": "git+https://github.com/cloudflare/cloudflare-docs-engine.git#1a796d3e"
+    "cloudflare-docs-engine": "git+https://github.com/cloudflare/cloudflare-docs-engine.git#1a796d3eaaa018da4c6ca4106bf369a2017bc4ec"
```
Comment (pedrosousa) [inline products/workers/package.json]:
```suggestion
    "cloudflare-docs-engine": "git+https://github.com/cloudflare/cloudflare-docs-engine.git"
```

**Review by eidam — APPROVED** [review]

**Comment by pedrosousa** [comment]
Already addressed in https://github.com/cloudflare/cloudflare-docs/pull/2811.

**Pull request closed without merging**